### PR TITLE
add doc for sysvar `tidb_session_alias`

### PR DESCRIPTION
**Pull request by lcwangchao**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v7.4 (TiDB 7.4 versions)
- [ ] v7.3 (TiDB 7.3 versions)
- [ ] v7.2 (TiDB 7.2 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs/pull/14725
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: system-variables.md [hunk 1]
```diff
@@ -3780,6 +3780,14 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 - 取值范围：`[128, 9223372036854775807]`，单位为 Byte。支持带单位的内存格式“KB|MB|GB|TB”。
 - 开启内存限制后，TiDB 会终止当前实例上内存用量最高的 SQL 语句。本变量指定此情况下 SQL 语句被终止的最小内存用量。如果 TiDB 实例的内存超限是由许多内存使用量不明显的会话导致的，可以适当调小该变量值，使得更多会话成为 Cancel 的对象。
 
+### `tidb_session_alias` <span class="version-mark">从 v7.4.0 版本开始引入</span>
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：字符串
+- 默认值：""
```
Comment (hfxsd) [inline system-variables.md:3788]: 需要加上可选值

Comment (lcwangchao) [reply system-variables.md:3788]: 它的值是可以随意指定的，只要符合格式限制。

File: system-variables.md [hunk 2]
```diff
@@ -3780,6 +3780,14 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
...
+- 用来自定义当前会话相关日志中 `session_alias` 列的值。此设置会对语句执行过程中多个节点的日志生效（包括 TiKV）。此变量限制长度最大为 64 个字符，且不能以空格结尾。
```
Comment (songrijie) [inline system-variables.md:3789]: 这个是不是已经改了？
```suggestion
- 用来自定义当前会话相关日志中 `session_alias` 列的值。此设置会对语句执行过程中多个节点的日志生效（包括 TiKV）。此变量限制长度最大为 64 个字符，末尾的空格会被截断。
```

**Comment by songrijie** [comment]
把这个方法在定位慢查询的介绍里也提一下吧 https://docs.pingcap.com/zh/tidb/stable/identify-slow-queries

File: system-variables.md (hunk 2 above)
Comment (lcwangchao) [reply system-variables.md:3789]: 其实超过 64 个字符也会被截断，是不是改成 “末尾的空格以及超出最大长度的部分会被截断” ？

Comment (songrijie) [reply system-variables.md:3789]:
```suggestion
- 用来自定义当前会话相关日志中 `session_alias` 列的值。此设置会对语句执行过程中多个节点的日志生效（包括 TiKV）。此变量限制长度最大为 64 个字符，超出的部分将会被自动截断。如果变量值的末尾存在空格，也会被自动去除。
```

**Review by songrijie — COMMENTED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

File: system-variables.md [hunk 3]
```diff
@@ -3780,6 +3780,14 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 - 取值范围：`[128, 9223372036854775807]`，单位为 Byte。支持带单位的内存格式“KB|MB|GB|TB”。
 - 开启内存限制后，TiDB 会终止当前实例上内存用量最高的 SQL 语句。本变量指定此情况下 SQL 语句被终止的最小内存用量。如果 TiDB 实例的内存超限是由许多内存使用量不明显的会话导致的，可以适当调小该变量值，使得更多会话成为 Cancel 的对象。
 
+### `tidb_session_alias` <span class="version-mark">从 v7.4.0 版本开始引入</span>
+
+- 作用域：SESSION
+- 是否持久化到集群：否
+- 类型：字符串
+- 默认值：""
+- 用来自定义当前会话相关日志中 `session_alias` 列的值。此设置会对语句执行过程中多个节点的日志生效（包括 TiKV）。此变量限制长度最大为 64 个字符，超出的部分将会被自动截断。如果变量值的末尾存在空格，也会被自动去除。
```
Comment (qiancai) [inline system-variables.md:3789]:
```suggestion
- 用来自定义当前会话相关日志中 `session_alias` 列的值，方便故障定位时识别该会话。此设置会对语句执行过程中涉及的多个节点的日志生效（包括 TiKV）。此变量限制长度最大为 64 个字符，超出的部分将会被自动截断。如果变量值的末尾存在空格，也会被自动去除。
```

**Review by qiancai — APPROVED** [review]

File: system-variables.md [hunk 4]
```diff
@@ -3780,6 +3780,14 @@ EXPLAIN FORMAT='brief' SELECT COUNT(1) FROM t WHERE a = 1 AND b IS NOT NULL;
 - 取值范围：`[128, 9223372036854775807]`，单位为 Byte。支持带单位的内存格式“KB|MB|GB|TB”。
 - 开启内存限制后，TiDB 会终止当前实例上内存用量最高的 SQL 语句。本变量指定此情况下 SQL 语句被终止的最小内存用量。如果 TiDB 实例的内存超限是由许多内存使用量不明显的会话导致的，可以适当调小该变量值，使得更多会话成为 Cancel 的对象。
 
+### `tidb_session_alias` <span class="version-mark">从 v7.4.0 版本开始引入</span>
+
+- 作用域：SESSION
+- 是否持久化到集群：否
```
Comment (Oreoxmt) [inline system-variables.md:3786]:
```suggestion
- 是否持久化到集群：否
- 是否受 Hint [SET_VAR](/optimizer-hints.md#set_varvar_namevar_value) 控制：否
```
@winoros PTAL

**Pull request merged**